CONTRIBUTING.md

```diff
@@ -33,9 +35,12 @@ $ mvn install
 #### Building Java Binding App
 Run:
 ```$sh
-$ cd exonum-java-binding-core/rust/ejb-app
-$ cargo install --debug --force
+$ ./package_app.sh
 ```
+This command will build and package EJB App with all the necessary runtime dependencies
+in a single `zip` archive in `exonum-java-binding-core/target` directory.
+
+**Automatic EJB App packaging is only supported on Linux platform.**
```

Comment:
Shan't we separate instructions for Linux and MacOS (i.e., add new for Linux and keep the old for MacOS)? Is there a way to launch a Java service on Mac with this PR integrated?

Reply:
Mac OS is not supported unfortunately.

Reply:
I would like to merge this PR in a separate branch (ejb-app-configuration), then make it working on Mac OS at least (and maybe Windows) - and then push it to master with working solution. 
Nevertheless, we can merge this into master. But "Java services can only be launched on Linux" is not completely true. I'm pretty sure you can launch EJB App on Mac OS manually by doing all the stuff that makes `package_app.sh`.

Reply:
> Nevertheless, we can merge this into master.

With no documentation and tests of it working on MacOS — I don't think so.